### ISSUE TITLE
Maintenance of this package

### ISSUE THREAD

**Issue by cpcloud** [conda-forge/pysuerga-feedstock#1]
### Comment:

I am unwilling to maintain this package anymore.

cc @datapythonista, if you're willing that'd be swell, otherwise we're going to archive this package.

cc @mariusvniekerk

**Comment by datapythonista** [comment]
Fine with me to leave it as is, so the current version is available via conda-forge, or to archive it if there is any advantage to you. Up to you.

**Comment by cpcloud** [comment]
Right, the issue is that I am the only maintainer and I do not wish to maintain the package anymore. One option is for you to become the maintainer. If you don't want to do that we should archive the repo. The conda package won't go away.

**Comment by datapythonista** [comment]
I think it's fine to archive the repo, and even more if the package is not removed with it.
